randovania/games/prime3/logic_database/Bryyo - Fire.txt

```diff
@@ -246,13 +239,33 @@ Extra - asset_id: 691790544662692104
               Bomb/Spring Space Jump (Expert)
               # https://youtu.be/_sSWzSqM_0g
               Screw Attack and Bomb/Spring Space Jump (Advanced)
+  > Ledge by Refinery Access Door
+      Any of the following:
+          Morph Ball Bomb and Morph Ball and Plasma Beam
+          All of the following:
+              Space Jump Boots
+              Any of the following:
+                  # https://youtu.be/NLcBG2JOZ9I
+                  Scan Visor and Movement (Intermediate)
+                  # https://youtu.be/66_fmkS3vOk
+                  Bomb/Spring Space Jump (Intermediate)
 
 > Event - Pipe; Heals? False
   * Layers: default
   * Event Gel Refinery Site Pipe
   > Door to Gel Purification Site
       Trivial
 
+> Ledge by Refinery Access Door; Heals? False
+  * Layers: default
+  > Door to Refinery Access
+      Plasma Beam
+  > Door to Gel Purification Site
+      # https://youtu.be/66_fmkS3vOk
+      Screw Attack at Z-Axis (Intermediate) and Slope Jump (Beginner) and Use Screw Attack (Space Jump)
```

Comment:
I don't agree with using this trick, as it's intended for NSJ Screw Attack. I'd just throw it under Movement.